frontend/views/components/GroupSettings/InviteLinks.vue

```diff
@@ -67,7 +64,7 @@ page-section.c-section(:title='L("Invite links")')
                 )
                   i18n See original proposal
```

Comment:
Leave a TODO here so we don't forget to make this menu actually work. It's something you can also do in a next PR :D

@mmbotelho what is the exact view I should see when clicking at "See original proposal"?

Reply:
I will work on it in next PR definitely.

Reply:
@sandrina-p it should take the user to the "Proposals" modal and show the proposal where the link was originally generated